lib/cdo/mailjet.rb

```diff
@@ -0,0 +1,110 @@
+require 'mailjet'
+require 'honeybadger/ruby'
+require_relative './shared_constants/mailjet_constants'
+
+module MailJet
+  include MailJetConstants
+
+  API_KEY = CDO.try(:mailjet_api_key).freeze
+  SECRET_KEY = CDO.try(:mailjet_secret_key).freeze
+
+  def self.enabled?
+    return false unless DCDO.get('use_mailjet', false)
+    !API_KEY.nil? && !SECRET_KEY.nil?
+  end
+
+  def self.subaccount
+    case Rails.env
+    when 'production'
+      'production'
+    when 'staging'
+      'staging'
+    else
+      'development'
+    end
+  end
```

Comment:
I was re-reading the tech plan and was reminded that we're explicitly planning on NOT connecting to mailjet for Test and Drone. Is there a good place to make that intention clear in the code?

Reply:
Yeah -- I could add a check in `enabled?`. Let me do that.

Reply:
oh, sorry, I meant clear to future developers wondering why production and staging are getting values but not anything else (because we don't have a standard convention or clear guidance about the different environments)

Reply:
oh gotcha! Yeah I can add something here as well